sql/src/parser.rs

```diff
@@ -216,14 +216,14 @@ impl<'a> Parser<'a> {
     }
 
     pub fn parse_show(&mut self) -> Result<Statement> {
-        if self
-            .parser
-            .parse_one_of_keywords(&[Keyword::CREATE])
-            .is_some()
-        {
+        if self.consume_token("TABLES") {
```

Comment:
Shall we define these token as const?

Reply:
These characters will only be used here, I don't think they need to be used as const.🤨And for reading, `show` -> `TABLES`/`DATABASES` will maybe be easier to understand then `show` -> TABLE_CONST.

Reply:
Sounds resonable.